drivers/led_strip/lpd880x.c

```diff
@@ -125,16 +139,16 @@ static int lpd880x_strip_init(struct device *dev)
 	struct lpd880x_data *data = dev->driver_data;
 	struct spi_config *config = &data->config;
 
-	data->spi = device_get_binding(CONFIG_LPD880X_STRIP_SPI_DEV_NAME);
+	data->spi = device_get_binding(DT_COLORWAY_LPD880X_0_BUS_NAME);
 	if (!data->spi) {
 		LOG_ERR("SPI device %s not found",
-			    CONFIG_LPD880X_STRIP_SPI_DEV_NAME);
+			    DT_COLORWAY_LPD880X_0_BUS_NAME);
 		return -ENODEV;
 	}
 
-	config->frequency = CONFIG_LPD880X_STRIP_SPI_BAUD_RATE;
+	config->frequency = DT_COLORWAY_LPD880X_0_SPI_MAX_FREQUENCY;
 	config->operation = LPD880X_SPI_OPERATION;
-	config->slave = 0;	/* MOSI/CLK only; CS is not supported. */
+	config->slave = DT_COLORWAY_LPD880X_0_BASE_ADDRESS; /* MOSI/CLK only; CS is not supported. */
```

Comment:
I don't get this part, sorry. This is the slave number, not an address.

Reply:
BASE_ADDRESS is the name we give to the reg field, for SPI devices the reg field is the slave address.